relay-general/src/store/transactions.rs

```diff
@@ -705,14 +713,36 @@ mod tests {
             ..Default::default()
         });
 
-        assert_eq_dbg!(
-            process_value(
-                &mut event,
-                &mut TransactionsProcessor::new(None),
-                ProcessingState::root()
-            ),
-            Err(ProcessingAction::InvalidTransaction("span is missing op"))
-        );
+        process_value(
+            &mut event,
+            &mut TransactionsProcessor::new(None),
+            ProcessingState::root(),
+        )
+        .unwrap();
+        assert_annotated_snapshot!(event, @r###"
+        {
+          "type": "transaction",
+          "timestamp": 946684810,
+          "start_timestamp": 946684800,
+          "contexts": {
+            "trace": {
+              "trace_id": "4c79f60c11214eb38604f4ae0781bfb2",
+              "span_id": "fa90fdead5f74053",
+              "op": "http.server",
+              "type": "trace"
+            }
+          },
+          "spans": [
+            {
+              "timestamp": 946684810,
+              "start_timestamp": 946684800,
+              "op": "op",
+              "span_id": "fa90fdead5f74053",
+              "trace_id": "4c79f60c11214eb38604f4ae0781bfb2"
+            }
+          ]
+        }
+        "###);
```

Comment:
Same as above, when the tests run, all timestamps in event reflect the current timestamp, not 2000-01-01 00:00:00 as set in the tests 😕